ZConfig/components/logger/tests/test_logger.py

```diff
@@ -27,11 +27,8 @@
 from ZConfig.components.logger import handlers
 from ZConfig.components.logger import loghandler
 
-try:
-    import StringIO as StringIO
-except ImportError:
-    # Python 3 support.
-    import io as StringIO
+from ZConfig._compat import NStringIO as StringIO
```

Comment:
It seemed like a slightly smaller change, but I admit it wasn't fully thought through. There was some evolution here. It can still change if desired.